### PR TITLE
crypto-square: apply lowerCamelCase policy

### PR DESCRIPTION
**Pull request by rpottsoh**
per #987

### CODE REVIEW

File: exercises/crypto-square/canonical-data.json [hunk 1]
```diff
@@ -1,60 +1,60 @@
 {
   "exercise": "crypto-square",
-  "version": "3.2.0",
+  "version": "3.2.1",
   "cases": [
     {
-      "description": "empty plaintext results in an empty ciphertext",
-      "property": "ciphertext",
+      "description": "empty plainText results in an empty cipherText",
```
Comment (ErikSchierboom) [inline exercises/crypto-square/canonical-data.json]: I assume you deliberately chose to use the property names, but I think it would maybe be better to use `plain text` and `cipher text` here.

Comment (rpottsoh) [reply exercises/crypto-square/canonical-data.json]: fixed descriptions to not use property names in text.

**Comment by rpottsoh** [comment]
Not at all @petertseng.  Thanks for feedback.

**Comment by rpottsoh** [comment]
@petertseng you were only referring to their use in the descriptions right?

File: exercises/crypto-square/canonical-data.json [hunk 2]
```diff
@@ -1,60 +1,60 @@
 {
   "exercise": "crypto-square",
-  "version": "3.2.0",
+  "version": "3.2.1",
   "cases": [
     {
       "description": "empty plaintext results in an empty ciphertext",
-      "property": "ciphertext",
+      "property": "cipherText",
```
Comment (petertseng) [inline exercises/crypto-square/canonical-data.json]: I think ciphertext is a better property than cipherText, so I would revert this change.

File: exercises/crypto-square/canonical-data.json [hunk 3]
```diff
@@ -1,60 +1,60 @@
...
       "input": {
-        "plaintext": ""
+        "plainText": ""
```
Comment (petertseng) [inline exercises/crypto-square/canonical-data.json]: I think plaintext is a better input key than plainText, so I would revert this change.

File: exercises/crypto-square/canonical-data.json [hunk 4]
```diff
@@ -1,60 +1,60 @@
 {
   "exercise": "crypto-square",
-  "version": "3.2.0",
+  "version": "3.2.1",
   "cases": [
     {
       "description": "empty plaintext results in an empty ciphertext",
```
Comment (petertseng) [inline exercises/crypto-square/canonical-data.json]: I think this is good. I think "plaintext" and "ciphertext" are better in descriptions than "plain text" and "cipher text"

**Comment by rpottsoh** [comment]
Closing this PR.  Citing https://en.wikipedia.org/wiki/Plaintext.  For this exercise "plaintext" and "ciphertext" are treated as single words.
